frontend/src/components/NewBoardForm/NewBoardForm.jsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { useState } from "react";
+import "./newboardform.css";
+
+export default function NewBoardForm({ setNewBrdFormOpened }) {
+  const [boardTitle, setBoardTitle] = useState("");
+  const [boardCat, setBoardCat] = useState("");
```

Comment:
Try to avoid contractions unless they're too obvious, is better to do the code more readable